book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/common/entity/SeatInShow.java

```diff
@@ -9,19 +9,25 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.experimental.SuperBuilder;
 import org.lbcc.bms.bms_monolith.common.enums.BookingStatus;
 
 @Entity
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
+@SuperBuilder
 public class SeatInShow extends BaseAuditingEntity {
 
     @ManyToOne
     @JoinColumn(name = "seat_type_in_show_id")
     private SeatTypeInShow seatTypeInShow;
 
+    @ManyToOne //fetch = FetchType.LAZY
+    @JoinColumn(name = "seat_id", nullable = false)
+    private Seat seat;
```

Comment:
This may lead to a Hibernate N+1 query problem.

Reply:
have tried to resolve this with eager fetching , please confirm if this solves the issue